expr/functions/seriesList/function.go

```diff
@@ -43,7 +43,11 @@ func (f *seriesList) Do(ctx context.Context, e parser.Expr, from, until int64, v
 		return nil, err
 	}
 
-	numerators, err := helper.GetSeriesArg(ctx, e.Args()[0], from, until, values)
+	if e.ArgsLen() < 2 {
+		return nil, types.ErrNeedMoreArguments
+	}
```

Comment:
This is already being done at the top of the function (though we used the `parser.ErrMissingArgument` error)

Reply:
Oh, I see panic on older version. Fixed in main.

Reply:
Yeah it was a very recent fix https://github.com/go-graphite/carbonapi/pull/701